include/mata/nfa.hh

```diff
@@ -1104,12 +1240,20 @@ public:
      */
     const AutSequence& get_segments();
 
+    /**
+     * Get raw segment automata.
+     * @return A vector of segments for the segment automaton in the order from the left (initial state in segment automaton)
+     * to the right (final states of segment automaton) without trimming (the states are same as in the original automaton).
+     */
+    const AutSequence& get_segments_raw();
```

Comment:
Maybe rename to `get_untrimmed_segments`? The raw sounds weird (I know I used it in python binding, but I actually hate it).

Reply:
We could not come up with a better name when we implemented this. I agree with your suggestion. `untrimmed` describes what the user gets well enough and it is not important for the user to know how we use the segments later.